src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -127,6 +127,34 @@
           "to 100 effectively disables the shortcut.")                      \
           range(0,100)                                                      \
                                                                             \
+  product(uintx, ShenandoahReactiveSampleFrequencyHz, 10, EXPERIMENTAL,     \
```

Comment:
Minor nit: these are now `ShenandoahAdaptive*`?

Reply:
Yes, will fix this.